src/js/lumo.js

```diff
@@ -107,6 +107,22 @@ export function fileExists(filename: string): boolean {
   return false;
 }
 
+export function stat(filePath: string): fs.Stats {
+  return fs.statSync(filePath);
```

Comment:
this could just be `export const stat = fs.statSync`. would avoid the extra layer of indirection